example/lib/main.dart

```diff
@@ -31,7 +33,22 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   setupNotifications();
   await LMChatCore.instance.initialize(
-    apiKey: "YOUR-API-KEY",
+    lmChatCallback: LMChatCoreCallback(
+        onAccessTokenExpiredAndRefreshed: (accessToken, refreshToken) async {
```

Comment:
add comments

Reply:
no need to add comment here, as it is in example layer, and it was written only to test `showChatWithoutApiKey()` flow, removed in final PR